sfge/src/main/java/com/salesforce/rules/unusedmethod/RuleStateTracker.java

```diff
@@ -196,11 +210,174 @@ List<String> getSubclasses(String... definingTypes) {
                                     .out(Schema.EXTENDED_BY));
             List<String> subclassNames =
                     subclassVertices.stream()
-                            .map(UserClassVertex::getName)
+                            .map(UserClassVertex::getDefiningType)
                             .collect(Collectors.toList());
             this.subclassesByDefiningType.put(definingType, subclassNames);
             results.addAll(subclassNames);
         }
         return results;
     }
+
+    /**
+     * Get any inner classes that reside in {@code definingType}.
+     *
+     * @param definingType
+     * @return
```

Comment:
While generating this as javadoc, the `@param` and `@return` would show up as empty sections. If we don't plan to populate them, can we remove empty/unused sections?
Sadly, Intellij generates them automatically and it's a pain to remember to clean them up.